prefab/dependency-integration/README.md

```diff
@@ -0,0 +1,87 @@
+Integrating Prefab Dependency
+==============================
+
+This sample shows how to import [C/C++ dependencies] from Google Maven to your Android project.
+Those dependencies are C/C++ libraries packed with [Prefab] format into an AAR.
+
+[Prefab] could be enabled from Android Gradle Plugin 4.0+. In roder to use the feature,
```

Comment:
Prefab is supported beginning with AGP 4.0.

To use the feature, follow these three steps: